### PR TITLE
docs:mkdocs.md doc chinese version

### PR DESCRIPTION
**Pull request by Black-Gold**
Signed-off-by: dingweiwei <dingweiwei@admin.com>

## Pre-Checklist

Note: please complete **_ALL_** items in the following checklist.

- [ ] I have read through the [CONTRIBUTING.md](https://github.com/devstream-io/devstream/blob/main/CONTRIBUTING.md) documentation.
- [ ] My code has the necessary comments and documentation (if needed).
- [ ] I have added relevant tests

### CODE REVIEW

File: docs/development/mkdocs.zh.md [hunk 1]
```diff
@@ -1,3 +1,69 @@
-# Creating a Documentation for DevStream 中文文档
+# 为DevStream创建中文文档
 
-todo
+## 背景
+
+1. 我们使用[readthedocs](https://readthedocs.org/)托管文档，不过您并不需要对readthedocs了解很多即可为DevStream创建文档。
```
Comment (daniel-hutao) [inline docs/development/mkdocs.zh.md:5]: Please use "你" here, not "您".

File: docs/development/mkdocs.zh.md [hunk 2]
```diff
@@ -1,3 +1,69 @@
...
+2. Readthedocs支持[`sphinx`](https://docs.readthedocs.io/en/stable/intro/getting-started-with-sphinx.html)和[`mkdocs`](https://docs.readthedocs.io/en/stable/intro/getting-started-with-mkdocs.html)来构建文档站点；我们现在使用`mkdocs`。如果您遇到任何问题，`mkdocs`的官方文档[这里](https://www.mkdocs.org/)是一个很好的起点。
+3. 在我们的mkdocs设置中，我们使用`material`主题。有关”material”的更多信息：
+    - [网站](https://squidfunk.github.io/mkdocs-material/)
+    - [文档](https://squidfunk.github.io/mkdocs-material/getting-started/)
+    - [GitHub 仓库](https://github.com/squidfunk/mkdocs-material)
+4. 我们还为mkdocs使用了两个插件：
+    - [search](https://squidfunk.github.io/mkdocs-material/setup/setting-up-site-search/)
```
Comment (daniel-hutao) [inline docs/development/mkdocs.zh.md:12]: search - 搜索

**Comment by IronCore864** [comment]
@Black-Gold Thanks for your contribution! We will test your change locally and try to merge it as soon as possible.

Regarding commit messages, we try our best to follow the [conventional commits specification](https://www.conventionalcommits.org/en/v1.0.0/) so that it would give other people clear information. We've also got a "TL;DR" version in our [documentation](https://docs.devstream.io/en/latest/development/commit-messages/); feel free to have a quick glance. Please provide useful information in your commit message; and please do not use duplicated commit messages.

You do not need to update your commit messages for this PR. We will use squash merge and update the commit messages accordingly. Please try to follow the standard next time :)

**Review by IronCore864 — APPROVED** [review]

**Comment by IronCore864** [comment]
To merge your PR quickly, I have resolved the comments and made an extra commit directly to your PR. I hope you don't mind. @Black-Gold Thanks for the contribution!

**Comment by IronCore864** [comment]
Regarding commit messages, we try our best to follow the [conventional commits specification](https://www.conventionalcommits.org/en/v1.0.0/) so that it would give other people clear information. We've also got a "TL;DR" version in our [documentation](https://docs.devstream.io/en/latest/development/commit-messages/); feel free to have a quick glance. Please provide useful information in your commit message; and please do not use duplicated commit messages.

You do not need to update your commit messages for this PR. We will use squash merge and update the commit messages accordingly. Please try to follow the standard next time :)

**Pull request merged**